app/styles/_core.less

```diff
@@ -978,6 +978,16 @@ a.disabled-link {
     margin-top: 0px;
     margin-bottom: 10px;
   }
+  .indent {
+    padding-left: 48px;
+    .title {
+      font-weight: 600;
+    }
+  }
+  .select-project-container {
+    padding-left: 20px;
+    width: 70%;
```

Comment:
How does this look at mobile?

Better to give it a max width in px instead of a percentage width. Then it can be 100% at mobile.